### PR TITLE
[Timeline] Update drag-over highlights promptly

### PR DESCRIPTION
**Pull request by VWoeltjen**
Invoke `$apply` from `mct-swimlane-drop` and add some checks to ensure this is necessary, to avoid
triggering a ton of digest cycles while dragging.

### Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? Y
3. Command line build passes? Y
4. Changes have been smoke-tested? Y

### CODE REVIEW

**Comment by VWoeltjen** [comment]
Addresses #767

File: platform/features/timeline/src/directives/MCTSwimlaneDrop.js [hunk 1]
```diff
@@ -97,21 +97,37 @@ define(
...
                 // Handle dragover
                 element.on('dragover', function (e) {
-                    dragOver(e, element, swimlane());
+                    var swimlane = lookupSwimlane(),
+                        highlight = swimlane.highlight(),
+                        highlightBottom = swimlane.highlightBottom();
+
+                    dragOver(e, element, swimlane);
+
+                    if ((highlightBottom !== swimlane.highlightBottom()) ||
```
Comment (larkin) [inline platform/features/timeline/src/directives/MCTSwimlaneDrop.js]: Unnecessary set of parenthesis in if statement.  Will push small correction.

**Comment by larkin** [comment]
@akhenry can you do a quick review on https://github.com/nasa/openmct/pull/782/commits/5e54b193ca54d848553c12c9f580ca8fe59fc34c?

**Comment by akhenry** [comment]
@larkin :+1:

**Comment by larkin** [comment]
# Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? Y
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y

**Pull request merged**